sdks/python/apache_beam/examples/wordcount_xlang_sql.py

```diff
@@ -15,12 +15,16 @@
 # limitations under the License.
 #
 
-"""A cross-language word-counting workflow."""
+"""A cross-language word-counting workflow.
+
+Java and docker must be available to run this pipeline.
```

Comment:
I think we need a README file more specific instructions for Dataflow and portable runners for users to easily try this out. I believe @TheNeuralBit is working on it.

Reply:
Yep - BEAM-10559 is tracking this.

Reply:
Sounds good.

Reply:
I don't know what such a readme would say. You run the pipeline with `python wordcount_xlang_sql.py` or `python -m apache_beam.examples.wordcount_xlang_sql` just as with all the other examples here. For Dataflow you set `--runner=DataflowRunner` (plus all the other parameters as documented elsewhere, but I did add a note about the experiment), for Flink you set `--runner=FlinkRunner`. Unlike kafka, there's no need for special instructions about setting up or connecting to an external cluster. Pretty much anything I can think of putting in a readme for this pipeline would be equally as well placed in a readme for running pipelines in general. 

The best readme is a readme that is not needed because it just works. 

(I do think it'd be good to have more docs and other more comprehensive examples, e.g. using `Row` and joins and more complicated queries.)

Reply:
Probably we need to include following information as well (either here on in a README).

(1) The fact that users have to install Java and which version to install
(2) Instructions for running from HEAD (for example, you need to push a Docker container to Docker Hub for Dataflow). May be you can just point to kafkataxi README for this.
(3) Example command for Dataflow with 'use_runner_v2' (and additionally sdk_harness_container_image_overrides for HEAD).

Reply:
That's true if I wrote a README, everything for running on Dataflow would just be copied from the kafkataxi README.

Reply:
At the top we note that Java is required. Good point about versions; I updated this to say java 8. I don't think we need instructions for building and running at head (which are not specific to SQL) in the very first introduction to using SqlTransform.

Reply:
Note that Docker is only required when running at HEAD (with a custom container).

Reply:
Updated the note about docker. It is required to run locally (ULR, local Flink cluster, ...). It's also required to create a custom container, but that's more about custom containers.